docs/customisation/workflow-customization.md

```diff
@@ -14,41 +14,65 @@ This is the root state machine for the application. On initialisation, it starts
 * vc.ts
 * settings.ts
 * activityLog.ts
-* request.ts
-* scan.ts
+* requestMachine.ts
+* scanMachine.ts
 * revoke.ts
 
 ## store.ts
 
-This state machine takes care of actions related to storing and retrieving data on the mobile phone. It exposes the wrapper to all the other state machines to work with data stored on the device. It also performs the custom encryption and decryption required for saving and retrieving data from the underlying store. As of now, the store state machine uses two libraries:
+This state machine takes care of actions related to storing and retrieving data on the mobile phone. It exposes the wrapper to all the other state machines to work with data stored on the device.
+
+It also performs the custom encryption and decryption required for saving and retrieving data from the underlying store. 
+
+As of now, the store state machine uses following two libraries which abstracts how the data is stored between iOS and Android:
 * [react-native-mmkv-storage](https://github.com/ammarahm-ed/react-native-mmkv-storage)  - stores all the meta information and references of the encrypted VC.
 * [react-native-fs](https://www.npmjs.com/package/react-native-fs) - stores the encrypted VC as a separate file,
 
-which abstracts how the data is stored between iOS and Android.
 
 ## auth.ts
 
-This state machine helps to set up authentication for the application. It allows users to choose between biometric and passcode-based authentication on the first launch.
+This state machine helps to set up authentication for the application. It allows users to choose between biometric and passcode-based authentication on the first launch. It also updates the app state as authorized or unauthorized based on user action on first launch.
+
+On first launch, once the user selects language preference, and goes through intro sliders, user has been asked to choose the unlock method.
+
+After selecting the unlock method as passcode or biometric, the user is navigated to Home screen Once user is on home screen, app state is considered as authorized. Once a user logs out from settings, the state is considered as unauthorized.
 
 ## vc.ts
 
-This state machine helps to show a couple of screens related to VC. One is `My VCs` where all user-downloaded VCs will be shown. Another one is `Received VCs` where all received VCs can be shown.
+This state machine takes care of the VC received in Inji as a Wallet or Verifier.
+When Inji is being used as a Wallet, all the user-downloaded VCs are displayed on the Home screen.
+
+It also keeps track of sharing VC over BLE. When Inji is being used as a Verifier, all received VCs are displayed under the Received Card option in Settings.
 
-## vcItem.ts
+## ExistingMosipVCItemMachine.ts
 
-This state machine performs all the VC-related activity for the current selected VC, like:
+This state machine takes care of VC downloaded via UIN/VID. This contains all the activities/state related to VC like:
```

Comment:
```suggestion
This state machine takes care of VC downloaded via OTP based flow. This contains all the activities/state related to VC like:
```

lets check in with `Swati P` on the correct business term here.

Reply:
I used UIN/VID as on UI also we also display Download via UIN/VID

Reply:
Will add these comment in PR raised against mosip org